### PR TITLE
Edit CLI usage for consistency, spelling, grammar, etc.

### PR DESCRIPTION
**Pull request by chenette**
Made usage text consistent by starting option descriptions with an imperative verb, "positional arguments" with a noun, and subcommand descriptions with an initial-capped verb in 3rd person singular.

Note: For the xo parent command, deleted the subcommand-only options (moved them into the subcommands).

Also added the missing "xo delete" subcommand to CLI reference doc.

### CODE REVIEW

File: sdk/examples/xo_python/sawtooth_xo/xo_cli.py [hunk 1]
```diff
@@ -93,73 +118,180 @@ def add_create_parser(subparsers, parent_parser):
...
         const=sys.maxsize,
         type=int,
-        help='wait for take to commit, set an integer to specify a timeout')
-
+        help='set time, in seconds, to wait for take transaction to commit')
 
 def add_delete_parser(subparsers, parent_parser):
     parser = subparsers.add_parser('delete', parents=[parent_parser])
 
+# def add_delete_parser(subparsers, parent_parser):
```
Comment (nick-drozd) [inline sdk/examples/xo_python/sawtooth_xo/xo_cli.py]: Remove commented code

File: sdk/examples/xo_python/sawtooth_xo/xo_cli.py [hunk 2]
```diff
@@ -93,73 +118,180 @@ def add_create_parser(subparsers, parent_parser):
...
+        type=str,
+        help='specify password for authentication if REST API is using Basic Auth')
 
     parser.add_argument(
         '--wait',
         nargs='?',
         const=sys.maxsize,
         type=int,
-        help='wait for take to commit, set an integer to specify a timeout')
+        help='set time, in seconds, to wait for take transaction to commit')
```
Comment (nick-drozd) [inline sdk/examples/xo_python/sawtooth_xo/xo_cli.py]: take -> delete

File: sdk/examples/xo_python/sawtooth_xo/xo_cli.py [hunk 3]
```diff
@@ -179,32 +311,32 @@ def create_parent_parser(prog_name):
...
-        type=str,
-        help='username for authentication if REST API is using Basic Auth')
-
-    parent_parser.add_argument(
-        '--auth-password',
-        type=str,
-        help='password for authentication if REST API is using Basic Auth')
+        help='display version information')
+
+#   parent_parser.add_argument(
```
Comment (nick-drozd) [inline sdk/examples/xo_python/sawtooth_xo/xo_cli.py]: commented code

File: sdk/examples/xo_python/sawtooth_xo/xo_cli.py (hunk 1 above)
Comment (chenette) [reply sdk/examples/xo_python/sawtooth_xo/xo_cli.py]: oops

**Pull request merged**